questions/244/hint.md

```diff
@@ -1 +1 @@
-Just pick the worst possible answer in the dropdown.
\ No newline at end of file
+The answer changed in C++23.
```

Comment:
I think this is a bit too vague. I suggest changing it to "The template is not instantiated" or something like that.